openTSNE/affinity.py

```diff
@@ -777,6 +795,7 @@ def __init__(
         n_jobs=1,
         random_state=None,
         verbose=False,
+        knn_kwargs=None,
```

Comment:
From what I can tell, you've added the `knn_kwargs` parameter to the `PerplexityBasedAffinities` and `MultiscaleMixture` classes. Could you please add them to the other Affinity types as well? This would certainly come in useful, e.g., in the `UniformAffinities` class. And I'd want it in the other affinity classes as well for consistency.

Reply:
Oh, I missed that, have updated the code.